### ISSUE TITLE
how to get fusion feature?

### ISSUE THREAD

**Issue by AnqiGou** [ChenyangSi/FreeU#57]
模型的输出层：
for i, module in enumerate(self.output_blocks):
            if only_mid_control or control is None:
                h = torch.cat([h, hs.pop()], dim=1)
            else:
                h = torch.cat([h, hs.pop() + control.pop()], dim=1)
            h = module(h, emb, context)

        h = h.type(x.dtype)
        return self.out(h)

请问论文中的fusion feature是直接拼接后的特征h = torch.cat([h, hs.pop() + control.pop()], dim=1)，还是再经过各层后的输出 h = module(h, emb, context)？
